wbb/modules/admin.py

```diff
@@ -342,3 +345,44 @@ async def unpin(_, message: Message):
             await app.unpin_all_chat_messages(chat_id)
     else:
         await message.reply_text("You're Not An Admin, Stop Spamming!")
+
+# Mute members
+
+
+@app.on_message(cust_filter.command(commands=("mute")) & ~filters.edited)
+async def mute(_, message: Message):
+    chat_id = message.chat.id
+    admins = await list_admins(chat_id)
+    from_user_id = message.from_user.id
+    username = message.text.replace("/mute", "")
+
+    if from_user_id in admins or from_user_id in SUDO:
+        if username != "":
+            victim = username
+        else:
+            victim = message.reply_to_message.from_user.id
+        await message.chat.restrict_member(victim,
+                                           permissions=ChatPermissions())
+        await message.reply_text("Muted!")
+    else:
+        await message.reply_text("Get Yourself An Admin Tag!")
+
+# Unmute members
```

Comment:
I don't think you need comment for this. but if you want to, use function docstrings instead.

Reply:
i used them everywhere